taf/developer_tool.py

```diff
@@ -95,6 +107,7 @@ def add_signing_key(
             num_of_signatures += 1
         if num_of_signatures == keys_num:
             all_loaded = True
+    update_roles(["snapshot", "timestamp"], taf_repo, keystore, role_infos)
```

Comment:
`keystore` could still be `None` here and I guess it will fail? This is fine for now, but when it comes to key rotation, I am not sure who will poses _snapshot_ and _timestamp_ keys. Probably we would use CI to sign those roles. I suggest  checking if `keystore is None` and skipping `update_roles` in that case, with a proper warning message.

Reply:
If `keystore is None` and these keys are not stored on Yubikeys, the users will be asked to directly enter these keys. This is handled by _load_signing_keys called from update_roles. Not sure if we'll need to modify this once we need to handle key rotation, but there won't be an error if `keystore is None` for now